SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -219,6 +219,24 @@ public class SwiftyTesseract {
     
     return data
   }
+
+  public func recognizedBlocksByLevel(_ level: ResultIteratorLevel) -> [RecognizedBlock] {
+
+    let tess_level = level.asTessarctLevel
+    guard let result_iterator = TessBaseAPIGetIterator(tesseract) else { return [] }
```

Comment:
A part of me wonders if a failure of getting the iterator should mean an empty array. Maybe the return type of this should be a `Result<[RecognizedBlock], Error>` and this line be a specific error that an iterator was unable to be created or something.

Reply:
Updated the return type.